README.md

```diff
@@ -1,6 +1,148 @@
+
+### Regole di gioco
+#### Rose
+* Ogni rosa deve essere composta da 25 giocatori, **obbligatoriamente** così suddivisi per ruoli:
+  * **3** Portieri
+  * **8** Difensori
+  * **8** Centrocampisti
+  * **6** Attaccanti
+  
+#### Formazioni
+* Termine ultimo per la consegna della formazione: **1 minuto**
+* Moduli consentiti: 
+ 	* 3-5-2
+ 	* 3-4-3
+ 	* 4-5-1
+ 	* 4-4-2
+ 	* 4-3-3
+ 	* 5-3-2
+ 	* 5-4-1
+* Se non inserita recupera l’ultima formazione salvata
+* Panchina con MAX 14 panchinari e nessun vincolo
+
+#### Calcolo
+* Bonus/Malus per eventi:
+	* Gol segnato: **+3**
+	* Gol subito: **-1**
+	* Rigore segnato: **+3**
+	* Rigore sbagliato: **-3**
+	* Ammonizione: **-0.5**
+	* Espulsione: **-1**
+	* Autogol: **-2**
+	* Assist soft: **+1**
+	* Assist: **+1**
+	* Assist gold: **+1**
+	* Gol decisivo pareggio: **0**
+	* Gol decisivo vittoria: **0**
+	* Portiere imbattuto: **+1**
+* Assegna voto ad ammonito s.v. **NO**
+* Fonte voti: **Fantacalcio**
+* Soglia gol: **66 punti**
+* Fasce gol: **ogni 6 (66, 72, 78, ...)**
+* Intorno all’interno di ciascuna fascia: **NO**
+* Intorno tra una fascia e l’altra: **NO**
+* Numero di sostituzioni consentite: **5**
+* Modalità sostituzioni: **Solo con cambio ruolo (nessun cambio modulo)**
+* Riserva d’ufficio: **SI fino al raggiungimento del numero di sostituzioni impostate**
+* Riserva d’ufficio per il portiere: **3 punti**
+
+#### Calcolo avanzato
+* Modificatore PORTIERE: **NO**
+* Modificatore DIFESA: **SI, includendo il portiere, con applicazione bonus/malus alla propria squadra**
+	* \< 6 ---> 0
+	* da 6 a 6.24 ---> +1
+	* da 6.25 a 6.49 ---> +2
+	* da 6.50 a 6.74 ---> +3
+	* da 6.75 a 6.99 ---> +4
+	* \>= 7 ---> +6
+* Modificatore CENTROCAMPO: **NO**
+* Modificatore ATTACCO: **NO**
+* Modificatore FAIRPLAY: **NO**
+* Modificatore RENDIMENTO: **NO**
+* Modificatore CAPITANO: **SI, per ogni singola partita**
+	* <= 6 ---> **0**
+	* 6.5 ---> **+0.5**
+	* 7 ---> **+1**
+	* >= 7.5 ---> **+1.5**
+* Il capitano raddoppia la somma dei bonus ottenuti: **NO**
+* Il capitano raddoppia la somma dei malus ottenuti: **NO**
+
+### Competizioni
+#### Campionato
+* Competizione **a calendario**
+* Struttura dei gironi: **Andata e ritorno più eventuale girone in campo neutro**
+* Inizio: prima giornata successiva all'asta estiva
+* Fattore campo: **+2** (escluso eventuale ultimo girone neutro)
+* Sequenza dei criteri di ordinamento classifica:
+	1. Punti
+	2. Somma punti totale
+	3. Gol fatti
+	4. Differenza reti
+	5. Classifica avulsa
+	6. Gol subiti
+	
+#### Coppa
+* Competizione **a gruppi**
+* Numero di gruppi: **2**
+* Composizione gruppi:
+	* Gruppo 1: **prime 4** in classifica di campionato al termine del girone di andata
+	* Gruppo 2: **ultime 4** in classifica di campionato al termine del girone di andata
```

Comment:
questo non dovrebbe essere 
girone A: 1^,3^,5^,7^
girone B: 2^,4^,6^,8^ 
?

altrimenti dal girone b escono necessariamente due squadre più scarse

Reply:
Quest'anno è stato così come scritto. Ricordo bene che abbiamo fatto in questo modo per dare più opportunità di vincere a chi è sotto in classifica, come incentivo per evitare che qualcuno col tempo smettesse di mettere la formazione / seguire

Reply:
Questo è comunque un regolamento di questa stagione. Ciò non toglie che potremmo modificarlo per il nuovo anno